Require a minimum confirmation count for beacon transactions (#338)
dcdpr/did-btc1: src/data-structures.md

- `versionId`: OPTIONAL ASCII string representation of the specific version of a DID document to be resolved.
- `versionTime`: OPTIONAL XML Datetime normalized to UTC without sub-second decimal precision. The DID document to be resolved is the most recent version of the DID document that was valid for the DID before the specified `versionTime`.

> - `minConf`: OPTIONAL positive integer (minimum `1`). The minimum number of Bitcoin block confirmations a [Beacon Signal] transaction requires to count towards resolution. Defaults to `6`.

Reviewer: ```suggestion
- `minConf`: OPTIONAL positive integer (minimum `1`). The minimum number of Bitcoin block confirmations required on a [Beacon Signal] transaction during resolution. Defaults to `6`.
```

I'm less confident about this rewording. I want to avoid the phrase "count towards" because it seems unusual for technical writing. I could see "considered for", "eligible for" and other variations. It feels like changing the sentence structure helps it read better without relying on "count".

My reasoning is that "count" is a mathematical term. The STE dictionary defines the word as "To add the number of objects or occurrences to get a total" and provides the approved example: "COUNT THE NUMBER OF TREADS ON THE TIRE." (On the other hand, the STE manual itself uses the phrase "counts as one word" 24 times. This suggests that I'm making a big deal over nothing.)

Author: lgtm. applying suggestion.
